src/test/java/seedu/address/logic/parser/UnassignCommandParserTest.java

```diff
@@ -0,0 +1,149 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.ASSIGNMENT_ID_DESC_ONE;
+import static seedu.address.logic.commands.CommandTestUtil.ASSIGNMENT_ID_DESC_TWO;
+import static seedu.address.logic.commands.CommandTestUtil.EMPLOYEE_ID_DESC_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_ASSIGNMENT_ID_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_EMPLOYEE_ID_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_PROJECT_ID_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.PREAMBLE_WHITESPACE;
+import static seedu.address.logic.commands.CommandTestUtil.PROJECT_ID_DESC_ALPHA;
+import static seedu.address.logic.commands.CommandTestUtil.PROJECT_ID_DESC_BETA;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_ASSIGNMENT_ID_ONE;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EMPLOYEE_ID_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PROJECT_ID_ALPHA;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ASSIGNMENT_ID;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMPLOYEE_ID;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROJECT_ID;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.UnassignCommand;
+import seedu.address.model.assignment.Assignment;
+import seedu.address.testutil.AssignmentBuilder;
+import seedu.address.testutil.TypicalPersons;
+import seedu.address.testutil.TypicalProjects;
+
+
+public class UnassignCommandParserTest {
```

Comment:
I like that you followed EP and included test cases for many different partitions